packages/trpc/server/routers/viewer/slots/util.ts

```diff
@@ -419,7 +420,7 @@ export async function getAvailableSlots({ input, ctx }: GetScheduleOptions): Pro
     }`
   );
   const getStartTime = (startTimeInput: string, timeZone?: string) => {
-    const startTimeMin = dayjs.utc().add(eventType.minimumBookingNotice || 1, "minutes");
+    const startTimeMin = dayjs.utc().add(1, "minutes");
```

Comment:
(MBN #1) Here, we use the backup case, so we can decide minimumBookingNotice further down